shuup/core/suppliers/base.py

```diff
@@ -33,6 +35,44 @@ def __init__(self, supplier, options):
         self.supplier = supplier
         self.options = options
 
+    def _can_handle_product(self, product_id):
```

Comment:
remove this from supplier class and move it to simple_supplier model

Reply:
But this is the simple_supplier model inherits from this class and all other supplier model should do that as well

Reply:
who said that? maybe the supplier model really doesn't care about the product internal type and checks for anything else